### PR TITLE
ci: adds workflow to store latest version of schemas in a folder

### LINKED ISSUE #670: Schemas to folder in project
# User story
As a user, I want to more easily test that the schema versions have been incremented, so I can no longer have to wait for github actions

# Acceptance criteria
- [ ] When a PR is merged a folder called schemas is created that stores JSON schemas
- [ ] The folder should contain only the latest versions

# Sprint Ready Checklist 
- [ ] 1. Acceptance criteria defined 
- [ ] 2. Team understands acceptance criteria 
- [ ] 3. Team has defined solution / steps to satisfy acceptance criteria 
- [ ] 4. Acceptance criteria is verifiable / testable 
- [ ] 5. External / 3rd Party dependencies identified
- [ ] 6. Ticket is prioritized and sized

# Notes
*Add any helpful notes here.*

### PR DESCRIPTION
**Pull request by yosefmaru**
Closes #670

### CODE REVIEW

**Review by jtyoung84 — CHANGES_REQUESTED** [review]
I just realized that this might  need be done in the tag workflow since the tag job needs to run after all things have been committed

**Comment by yosefmaru** [comment]
> I just realized that this might need be done in the tag workflow since the tag job needs to run after all things have been committed

That makes sense. I moved update_schemas job to the tag workflow. The tag job will need update_docs and update_schemas.

**Review by jtyoung84 — APPROVED** [review]

**Comment by jtyoung84** [comment]
@dyf @saskiad This pull request will automatically build and save the latest schemas to a folder called schemas in the root level of the project directory

File: .github/workflows/tag.yml [hunk 1]
```diff
@@ -32,8 +32,34 @@ jobs:
...
+          ref: ${{ env.DEFAULT_BRANCH }}
+          fetch-depth: 0
+          token: ${{ secrets.SERVICE_TOKEN }}
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.8
+      - name: Create schemas
+        run: | 
+          python -m pip install -e .
```
Comment (jtyoung84) [inline .github/workflows/tag.yml:52]: You may need a line to remove the contents of the old directory

**Review by jtyoung84 — CHANGES_REQUESTED** [review]
You may need a line to remove the contents of the old directory

**Review by jtyoung84 — APPROVED** [review]

**Pull request merged**